doc/decoding/going_further.rst

```diff
@@ -44,8 +44,8 @@ data to 2D arrays (shape (n_timepoints, n_voxels)) that estimators can work on.
 
 .. note::
 
-  This example shows how to use masking:
-    :ref:`sphx_glr_auto_examples_04_manipulating_images_/plot_nifti_simple.py`
+   This example shows how to use masking:
+   :ref:`sphx_glr_auto_examples_04_manipulating_images_/plot_nifti_simple.py`
```

Comment:
The latest push will fail because this link is not in correct form

```suggestion
   This example shows how to use masking:
   :ref:`sphx_glr_auto_examples_06_manipulating_images_plot_nifti_simple.py`
```